corehq/apps/export/const.py

```diff
@@ -14,7 +14,7 @@
 
 # When fixing a bug that requires existing schemas to be rebuilt,
 # bump the version number.
-DATA_SCHEMA_VERSION = 6
+DATA_SCHEMA_VERSION = 7
```

Comment:
interesting thought, i'm about to make another schema change so i'll put it into that